# HG changeset patch
# User Rik <rik@octave.org>
# Date 1452711893 28800
#      Wed Jan 13 11:04:53 2016 -0800
# Node ID d48fdf3a8c0c43ee3d134369ecc802aa2934ee04
# Parent  5e00ed38a58bd2d6cce46554210016e44ee23fb8
maint: one more addition to cset 5e00ed38a58b.

* CollocWt.cc: Replace if/error/else paradigm with just if/error.

diff --git a/liboctave/numeric/CollocWt.cc b/liboctave/numeric/CollocWt.cc
--- a/liboctave/numeric/CollocWt.cc
+++ b/liboctave/numeric/CollocWt.cc
@@ -436,20 +436,17 @@ CollocWt::init (void)
   A.resize (nt, nt);
   B.resize (nt, nt);
 
   double *pr = r.fortran_vec ();
 
   // Compute roots.
 
   if (! jcobi (n, inc_left, inc_right, Alpha, Beta, pdif1, pdif2, pdif3, pr))
-    {
-      error ("jcobi: newton iteration failed");
-      return;
-    }
+    error ("jcobi: newton iteration failed");
 
   octave_idx_type id;
 
   // First derivative weights.
 
   id = 1;
   for (octave_idx_type i = 0; i < nt; i++)
     {
